content/en/synthetics/apm.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Synthetics APM
+kind: documentation
+beta: true
+description: APM and Distributed Tracing with Synthetics
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "tracing/"
+  tag: "Documentation"
+  text: "APM and Distributed Tracing"
+
+---
+
+<div class="alert alert-warning">Synthetics is in beta for the Datadog US Site. To request access, complete the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics request form</a>.</div>
+
+{{< img src="synthetics/apm/synthetics-apm.gif" alt="APM and Synthetics" responsive="true" style="width:80%;">}}
+
+## Overview
+
+The APM integration with Synthetics allows you to go from a test run that potentially failed to the root cause of the issue by looking at the trace generated by that very test run.
+
+Having both network related details thanks to your test and backend information thanks to your trace allows you to access a whole new level of details in the way your application is behaving as experienced by your user.
```

Comment:
```suggestion
Having both network related specifics, thanks to your test, and backend information, thanks to your trace, allows you to access a whole new level of details in the way your application is behaving as experienced by your user.
```

Reply:
From GJ: Having both network related specifics, thanks to your test, and backend, infra and log information, thanks to your trace, allows you to access a whole new level of details in the way your application is behaving as experienced by your user.